### PR TITLE
Makes it possible to listen to mouse drop events

### PR DESCRIPTION
**Pull request by dhuebner**
Adds two new even types `dragover` and `drop`. 

Enhanced the Circle example to demonstrate the  drag and drop functionality.

Try it out by dragging the 'drag' circle into the Graph:
 
Bildschirmfoto 2022-08-26 um 15 20 24

Also added a new launch config to run all the tests.

### CODE REVIEW

**Comment by dhuebner** [comment]
Some internal Jenkins failure?

**Review by spoenemann — COMMENTED** [review]
Thanks! Minor observation: when I drag the example "drag" button, the upper part of the diagram starts following the mouse cursor as well:
Screenshot 2022-08-29 at 09 49 04 (2)

@planger any objections to adding these two event types to mouse listeners? Have you already dealt differently with drag & drop in the context of GLSP?

**Comment by planger** [comment]
@spoenemann Thanks for pinging me! I don't have any objects, nice work! :+1: 

In GLSP, there is an extended mouse listener which is ["dragaware"](https://github.com/eclipse-glsp/glsp-client/blob/master/packages/client/src/base/drag-aware-mouse-listener.ts), so we'll have to double-check if that causes any conflicts, but from looking at the code, I wouldn't expect any issues.

With GLSP, we handled drag and drop so far always in the Theia diagram widget, which then passes on a specific action to the actual diagram client. Outside of Theia, I think we never had this requirement so far.

Anyway, native support for drag and drop in Sprotty is for sure a good thing to have!

File: examples/circlegraph/src/di.config.ts [hunk 1]
```diff
@@ -17,19 +17,21 @@
...
+export default (nodeCreator: (point?: Point)=>void) => {
     require("sprotty/css/sprotty.css");
     require("../css/diagram.css");
 
     const circlegraphModule = new ContainerModule((bind, unbind, isBound, rebind) => {
         bind(TYPES.ModelSource).to(LocalModelSource).inSingletonScope();
         rebind(TYPES.ILogger).to(ConsoleLogger).inSingletonScope();
         rebind(TYPES.LogLevel).toConstantValue(LogLevel.log);
-
+        const dropHandler = new DroppableMouseListener(nodeCreator);
```
[the remark this answers is not in the document]
Comment (dhuebner) [reply examples/circlegraph/src/di.config.ts:33]: Wii try this

**Comment by dhuebner** [comment]
@spoenemann 
Thanks for the re-view!
> Minor observation: when I drag the example "drag" button, the upper part of the diagram starts following the mouse cursor as well:

That is strange... It only happens in Chrom. Safari and Firefox works well, I will look into it!

File: examples/circlegraph/src/di.config.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (dhuebner) [reply examples/circlegraph/src/di.config.ts]: Done

**Comment by dhuebner** [comment]
@spoenemann 
Now it also works well with Chrom. DI was also changed. Commit is rebased on top of master

**Review by spoenemann — APPROVED** [review]
Thanks!

**Pull request merged**